src/main/java/com/bakdata/kafka/util/ImprovedAdminClient.java

```diff
@@ -44,20 +44,19 @@ public final class ImprovedAdminClient implements Closeable {
 
     @Getter
     private final @NonNull Properties properties;
-    @Getter
     private final @NonNull AdminClient adminClient;
-    @Getter
```

Comment:
Why did you remove this? We should keep the getter but make it return an optional

Reply:
Good idea I'll add a getter